A lot of scripts have bashisms like local function vars, so let them use bash

diff -ur resource-agents-3.9.7-1/heartbeat/anything resource-agents-3.9.7/heartbeat/anything
--- resource-agents-3.9.7-1/heartbeat/anything	2017-02-18 12:22:23.015278333 +0300
+++ resource-agents-3.9.7/heartbeat/anything	2017-02-18 12:36:29.051367301 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #       OCF Resource Agent compliant resource script.
 #
diff -ur resource-agents-3.9.7-1/heartbeat/apache resource-agents-3.9.7/heartbeat/apache
--- resource-agents-3.9.7-1/heartbeat/apache	2017-02-18 12:22:23.026237007 +0300
+++ resource-agents-3.9.7/heartbeat/apache	2017-02-18 12:36:29.124081391 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	High-Availability Apache/IBMhttp control script
 # 
diff -ur resource-agents-3.9.7-1/heartbeat/apache.~1~ resource-agents-3.9.7/heartbeat/apache.~1~
--- resource-agents-3.9.7-1/heartbeat/apache.~1~	2017-02-18 12:22:23.015421309 +0300
+++ resource-agents-3.9.7/heartbeat/apache.~1~	2017-02-18 12:36:28.986999556 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	High-Availability Apache/IBMhttp control script
 # 
diff -ur resource-agents-3.9.7-1/heartbeat/asterisk resource-agents-3.9.7/heartbeat/asterisk
--- resource-agents-3.9.7-1/heartbeat/asterisk	2017-02-18 12:22:23.015676412 +0300
+++ resource-agents-3.9.7/heartbeat/asterisk	2017-02-18 12:36:29.172920390 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # Asterisk
diff -ur resource-agents-3.9.7-1/heartbeat/CTDB resource-agents-3.9.7/heartbeat/CTDB
--- resource-agents-3.9.7-1/heartbeat/CTDB	2017-02-18 12:22:23.008872764 +0300
+++ resource-agents-3.9.7/heartbeat/CTDB	2017-02-18 12:36:29.095413643 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #             OCF Resource Agent for managing CTDB
 #
diff -ur resource-agents-3.9.7-1/heartbeat/db2 resource-agents-3.9.7/heartbeat/db2
--- resource-agents-3.9.7-1/heartbeat/db2	2017-02-18 12:22:23.016149066 +0300
+++ resource-agents-3.9.7/heartbeat/db2	2017-02-18 12:36:29.136411355 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # db2
 #
diff -ur resource-agents-3.9.7-1/heartbeat/dhcpd resource-agents-3.9.7/heartbeat/dhcpd
--- resource-agents-3.9.7-1/heartbeat/dhcpd	2017-02-18 12:22:23.016308117 +0300
+++ resource-agents-3.9.7/heartbeat/dhcpd	2017-02-18 12:36:29.108733411 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #   Resource Agent for managing dhcpd resources.
 #
diff -ur resource-agents-3.9.7-1/heartbeat/docker resource-agents-3.9.7/heartbeat/docker
--- resource-agents-3.9.7-1/heartbeat/docker	2017-02-18 12:22:23.016649524 +0300
+++ resource-agents-3.9.7/heartbeat/docker	2017-02-18 12:36:29.142726111 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # The docker HA resource agent creates and launches a docker container
 # based off a supplied docker image. Containers managed by this agent
diff -ur resource-agents-3.9.7-1/heartbeat/ethmonitor resource-agents-3.9.7/heartbeat/ethmonitor
--- resource-agents-3.9.7-1/heartbeat/ethmonitor	2017-02-18 12:22:23.016956381 +0300
+++ resource-agents-3.9.7/heartbeat/ethmonitor	2017-02-18 12:36:29.073263277 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	   OCF Resource Agent compliant script.
 #	   Monitor the vitality of a local network interface.
diff -ur resource-agents-3.9.7-1/heartbeat/Evmsd resource-agents-3.9.7/heartbeat/Evmsd
--- resource-agents-3.9.7-1/heartbeat/Evmsd	2017-02-18 12:22:23.009829802 +0300
+++ resource-agents-3.9.7/heartbeat/Evmsd	2017-02-18 12:36:29.129261268 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	Evmsd OCF RA.
 #
diff -ur resource-agents-3.9.7-1/heartbeat/EvmsSCC resource-agents-3.9.7/heartbeat/EvmsSCC
--- resource-agents-3.9.7-1/heartbeat/EvmsSCC	2017-02-18 12:22:23.009684133 +0300
+++ resource-agents-3.9.7/heartbeat/EvmsSCC	2017-02-18 12:36:29.089050539 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Support:      linux-ha@lists.linux-ha.org
 # License:      GNU General Public License (GPL)
diff -ur resource-agents-3.9.7-1/heartbeat/exportfs resource-agents-3.9.7/heartbeat/exportfs
--- resource-agents-3.9.7-1/heartbeat/exportfs	2017-02-18 12:22:23.017111052 +0300
+++ resource-agents-3.9.7/heartbeat/exportfs	2017-02-18 12:36:29.160273275 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 # exportfs
 #
 # Description: Manages nfs exported file system.
diff -ur resource-agents-3.9.7-1/heartbeat/Filesystem resource-agents-3.9.7/heartbeat/Filesystem
--- resource-agents-3.9.7-1/heartbeat/Filesystem	2017-02-18 12:22:23.009974927 +0300
+++ resource-agents-3.9.7/heartbeat/Filesystem	2017-02-18 12:36:29.149823649 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Support:      linux-ha@lists.linux-ha.org
 # License:      GNU General Public License (GPL)
diff -ur resource-agents-3.9.7-1/heartbeat/findif.sh resource-agents-3.9.7/heartbeat/findif.sh
--- resource-agents-3.9.7-1/heartbeat/findif.sh	2017-02-18 12:22:23.017277348 +0300
+++ resource-agents-3.9.7/heartbeat/findif.sh	2017-02-18 12:36:28.959082485 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 ipcheck_ipv4() {
   local r1_to_255="([1-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])"
   local r0_to_255="([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])"
diff -ur resource-agents-3.9.7-1/heartbeat/galera resource-agents-3.9.7/heartbeat/galera
--- resource-agents-3.9.7-1/heartbeat/galera	2017-02-18 12:22:23.017615156 +0300
+++ resource-agents-3.9.7/heartbeat/galera	2017-02-18 12:36:28.944427557 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Copyright (c) 2014 David Vossel <davidvossel@gmail.com>
 #                    All Rights Reserved.
diff -ur resource-agents-3.9.7-1/heartbeat/ICP resource-agents-3.9.7/heartbeat/ICP
--- resource-agents-3.9.7-1/heartbeat/ICP	2017-02-18 12:22:23.010127502 +0300
+++ resource-agents-3.9.7/heartbeat/ICP	2017-02-18 12:36:29.011653772 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # 
 # ICP
diff -ur resource-agents-3.9.7-1/heartbeat/IPaddr resource-agents-3.9.7/heartbeat/IPaddr
--- resource-agents-3.9.7-1/heartbeat/IPaddr	2017-02-18 12:22:23.010301636 +0300
+++ resource-agents-3.9.7/heartbeat/IPaddr	2017-02-18 12:36:29.115833515 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # License:      GNU General Public License (GPL) 
 # Support:      linux-ha@lists.linux-ha.org
diff -ur resource-agents-3.9.7-1/heartbeat/IPaddr2 resource-agents-3.9.7/heartbeat/IPaddr2
--- resource-agents-3.9.7-1/heartbeat/IPaddr2	2017-02-18 12:22:23.010472690 +0300
+++ resource-agents-3.9.7/heartbeat/IPaddr2	2017-02-18 12:36:29.068573870 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	$Id: IPaddr2.in,v 1.24 2006/08/09 13:01:54 lars Exp $
 #
diff -ur resource-agents-3.9.7-1/heartbeat/IPsrcaddr resource-agents-3.9.7/heartbeat/IPsrcaddr
--- resource-agents-3.9.7-1/heartbeat/IPsrcaddr	2017-02-18 12:22:23.010609656 +0300
+++ resource-agents-3.9.7/heartbeat/IPsrcaddr	2017-02-18 12:36:28.973124372 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	Description:	IPsrcaddr - Preferred source address modification
 #
diff -ur resource-agents-3.9.7-1/heartbeat/iscsi resource-agents-3.9.7/heartbeat/iscsi
--- resource-agents-3.9.7-1/heartbeat/iscsi	2017-02-18 12:22:23.018750741 +0300
+++ resource-agents-3.9.7/heartbeat/iscsi	2017-02-18 12:36:28.948678091 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # iSCSI OCF resource agent
 # Description: manage iSCSI disks (add/remove) using open-iscsi
diff -ur resource-agents-3.9.7-1/heartbeat/jboss resource-agents-3.9.7/heartbeat/jboss
--- resource-agents-3.9.7-1/heartbeat/jboss	2017-02-18 12:22:23.018877683 +0300
+++ resource-agents-3.9.7/heartbeat/jboss	2017-02-18 12:36:29.048704642 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Description:  Manages a Jboss Server as an OCF High-Availability
 #               resource under Heartbeat/LinuxHA control
diff -ur resource-agents-3.9.7-1/heartbeat/LVM resource-agents-3.9.7/heartbeat/LVM
--- resource-agents-3.9.7-1/heartbeat/LVM	2017-02-18 12:22:23.011034605 +0300
+++ resource-agents-3.9.7/heartbeat/LVM	2017-02-18 12:36:28.978982802 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # 
 # LVM
diff -ur resource-agents-3.9.7-1/heartbeat/mysql resource-agents-3.9.7/heartbeat/mysql
--- resource-agents-3.9.7-1/heartbeat/mysql	2017-02-18 12:22:23.019361050 +0300
+++ resource-agents-3.9.7/heartbeat/mysql	2017-02-18 12:36:29.081747664 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # MySQL
diff -ur resource-agents-3.9.7-1/heartbeat/mysql-common.sh resource-agents-3.9.7/heartbeat/mysql-common.sh
--- resource-agents-3.9.7-1/heartbeat/mysql-common.sh	2017-02-18 12:22:23.019494985 +0300
+++ resource-agents-3.9.7/heartbeat/mysql-common.sh	2017-02-18 12:36:29.139033358 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 
 #######################################################################
 
diff -ur resource-agents-3.9.7-1/heartbeat/mysql-proxy resource-agents-3.9.7/heartbeat/mysql-proxy
--- resource-agents-3.9.7-1/heartbeat/mysql-proxy	2017-02-18 12:22:23.019657501 +0300
+++ resource-agents-3.9.7/heartbeat/mysql-proxy	2017-02-18 12:36:29.057263404 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Resource script for MySQL Proxy
 #
diff -ur resource-agents-3.9.7-1/heartbeat/named resource-agents-3.9.7/heartbeat/named
--- resource-agents-3.9.7-1/heartbeat/named	2017-02-18 12:22:23.019962773 +0300
+++ resource-agents-3.9.7/heartbeat/named	2017-02-18 12:36:29.164363160 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Description:  Manages a named (Bind) server as an OCF High-Availability
 #               resource
diff -ur resource-agents-3.9.7-1/heartbeat/nfsserver resource-agents-3.9.7/heartbeat/nfsserver
--- resource-agents-3.9.7-1/heartbeat/nfsserver	2017-02-18 12:22:23.020254453 +0300
+++ resource-agents-3.9.7/heartbeat/nfsserver	2017-02-18 12:36:28.936870154 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 # nfsserver
 #
 # Description: Manages nfs server as OCF resource
diff -ur resource-agents-3.9.7-1/heartbeat/nginx resource-agents-3.9.7/heartbeat/nginx
--- resource-agents-3.9.7-1/heartbeat/nginx	2017-02-18 12:22:23.020514543 +0300
+++ resource-agents-3.9.7/heartbeat/nginx	2017-02-18 12:36:29.038747637 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	High-Availability nginx OCF resource agent
 # 
diff -ur resource-agents-3.9.7-1/heartbeat/oracle resource-agents-3.9.7/heartbeat/oracle
--- resource-agents-3.9.7-1/heartbeat/oracle	2017-02-18 12:22:23.021626562 +0300
+++ resource-agents-3.9.7/heartbeat/oracle	2017-02-18 12:36:29.031136581 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # 
 # oracle
diff -ur resource-agents-3.9.7-1/heartbeat/oralsnr resource-agents-3.9.7/heartbeat/oralsnr
--- resource-agents-3.9.7-1/heartbeat/oralsnr	2017-02-18 12:22:23.021815045 +0300
+++ resource-agents-3.9.7/heartbeat/oralsnr	2017-02-18 12:36:29.059693050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # 
 # oralsnr
@@ -34,7 +34,7 @@
 
 #######################################################################
 
-SH=/bin/sh
+SH=/usr/bin/bash
 
 oralsnr_usage() {
   methods=`oralsnr_methods`
diff -ur resource-agents-3.9.7-1/heartbeat/pgsql resource-agents-3.9.7/heartbeat/pgsql
--- resource-agents-3.9.7-1/heartbeat/pgsql	2017-02-18 12:22:23.022023589 +0300
+++ resource-agents-3.9.7/heartbeat/pgsql	2017-02-18 12:36:29.009095527 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Description:  Manages a PostgreSQL Server as an OCF High-Availability
 #               resource
diff -ur resource-agents-3.9.7-1/heartbeat/portblock resource-agents-3.9.7/heartbeat/portblock
--- resource-agents-3.9.7-1/heartbeat/portblock	2017-02-18 12:22:23.022358278 +0300
+++ resource-agents-3.9.7/heartbeat/portblock	2017-02-18 12:36:29.020461636 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # portblock: iptables temporary portblocking control 
 #
diff -ur resource-agents-3.9.7-1/heartbeat/postfix resource-agents-3.9.7/heartbeat/postfix
--- resource-agents-3.9.7-1/heartbeat/postfix	2017-02-18 12:22:23.022544636 +0300
+++ resource-agents-3.9.7/heartbeat/postfix	2017-02-18 12:36:28.968825553 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Resource script for Postfix
 #
diff -ur resource-agents-3.9.7-1/heartbeat/pound resource-agents-3.9.7/heartbeat/pound
--- resource-agents-3.9.7-1/heartbeat/pound	2017-02-18 12:22:23.022693772 +0300
+++ resource-agents-3.9.7/heartbeat/pound	2017-02-18 12:36:29.118649259 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # Pound
diff -ur resource-agents-3.9.7-1/heartbeat/Pure-FTPd resource-agents-3.9.7/heartbeat/Pure-FTPd
--- resource-agents-3.9.7-1/heartbeat/Pure-FTPd	2017-02-18 12:22:23.011966719 +0300
+++ resource-agents-3.9.7/heartbeat/Pure-FTPd	2017-02-18 12:36:29.097585284 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Resource script for Pure-FTPd
 #
diff -ur resource-agents-3.9.7-1/heartbeat/rabbitmq-cluster resource-agents-3.9.7/heartbeat/rabbitmq-cluster
--- resource-agents-3.9.7-1/heartbeat/rabbitmq-cluster	2017-02-18 12:22:23.023139258 +0300
+++ resource-agents-3.9.7/heartbeat/rabbitmq-cluster	2017-02-18 12:36:29.084935473 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Copyright (c) 2014 David Vossel <davidvossel@gmail.com>
 #                    All Rights Reserved.
diff -ur resource-agents-3.9.7-1/heartbeat/Raid1 resource-agents-3.9.7/heartbeat/Raid1
--- resource-agents-3.9.7-1/heartbeat/Raid1	2017-02-18 12:22:23.026083167 +0300
+++ resource-agents-3.9.7/heartbeat/Raid1	2017-02-18 12:36:29.043327271 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # License:      GNU General Public License (GPL)
diff -ur resource-agents-3.9.7-1/heartbeat/Raid1.~1~ resource-agents-3.9.7/heartbeat/Raid1.~1~
--- resource-agents-3.9.7-1/heartbeat/Raid1.~1~	2017-02-18 12:22:23.012425603 +0300
+++ resource-agents-3.9.7/heartbeat/Raid1.~1~	2017-02-18 12:36:29.104220228 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # License:      GNU General Public License (GPL)
diff -ur resource-agents-3.9.7-1/heartbeat/SAPInstance resource-agents-3.9.7/heartbeat/SAPInstance
--- resource-agents-3.9.7-1/heartbeat/SAPInstance	2017-02-18 12:22:23.012933844 +0300
+++ resource-agents-3.9.7/heartbeat/SAPInstance	2017-02-18 12:36:28.956678850 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # SAPInstance
 #
@@ -44,7 +44,7 @@
 
 #######################################################################
 
-SH=/bin/sh
+SH=/usr/bin/bash
 
 sapinstance_usage() {
   methods=`sapinstance_methods`
diff -ur resource-agents-3.9.7-1/heartbeat/SendArp resource-agents-3.9.7/heartbeat/SendArp
--- resource-agents-3.9.7-1/heartbeat/SendArp	2017-02-18 12:22:23.013131367 +0300
+++ resource-agents-3.9.7/heartbeat/SendArp	2017-02-18 12:36:29.087161930 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 #	Copyright (c) 2006, Huang Zhen <zhen.huang@gmail.com>
diff -ur resource-agents-3.9.7-1/heartbeat/SphinxSearchDaemon resource-agents-3.9.7/heartbeat/SphinxSearchDaemon
--- resource-agents-3.9.7-1/heartbeat/SphinxSearchDaemon	2017-02-18 12:22:23.013497020 +0300
+++ resource-agents-3.9.7/heartbeat/SphinxSearchDaemon	2017-02-18 12:36:29.015903054 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 #	Searchd OCF RA. 
diff -ur resource-agents-3.9.7-1/heartbeat/tomcat resource-agents-3.9.7/heartbeat/tomcat
--- resource-agents-3.9.7-1/heartbeat/tomcat	2017-02-18 12:22:23.025341968 +0300
+++ resource-agents-3.9.7/heartbeat/tomcat	2017-02-18 12:36:28.965171770 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Description:  Manages a Tomcat Server as an OCF High-Availability
 #               resource under Heartbeat/LinuxHA control
diff -ur resource-agents-3.9.7-1/heartbeat/varnish resource-agents-3.9.7/heartbeat/varnish
--- resource-agents-3.9.7-1/heartbeat/varnish	2017-02-18 12:22:23.025481181 +0300
+++ resource-agents-3.9.7/heartbeat/varnish	2017-02-18 12:36:29.127841651 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # Varnish
diff -ur resource-agents-3.9.7-1/heartbeat/VirtualDomain resource-agents-3.9.7/heartbeat/VirtualDomain
--- resource-agents-3.9.7-1/heartbeat/VirtualDomain	2017-02-18 12:22:23.014384064 +0300
+++ resource-agents-3.9.7/heartbeat/VirtualDomain	2017-02-18 12:36:29.156686827 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Support:	  linux-ha@lists.linux-ha.org
 # License:	  GNU General Public License (GPL)
diff -ur resource-agents-3.9.7-1/heartbeat/WAS resource-agents-3.9.7/heartbeat/WAS
--- resource-agents-3.9.7-1/heartbeat/WAS	2017-02-18 12:22:23.014540817 +0300
+++ resource-agents-3.9.7/heartbeat/WAS	2017-02-18 12:36:29.168893335 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # 
 # WAS
diff -ur resource-agents-3.9.7-1/heartbeat/WAS6 resource-agents-3.9.7/heartbeat/WAS6
--- resource-agents-3.9.7-1/heartbeat/WAS6	2017-02-18 12:22:23.014704317 +0300
+++ resource-agents-3.9.7/heartbeat/WAS6	2017-02-18 12:36:29.024993083 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 # WAS6 
 #
 # Description:  Manages a Websphere Application Server as an HA resource
diff -ur resource-agents-3.9.7-1/heartbeat/WinPopup resource-agents-3.9.7/heartbeat/WinPopup
--- resource-agents-3.9.7-1/heartbeat/WinPopup	2017-02-18 12:22:23.014861532 +0300
+++ resource-agents-3.9.7/heartbeat/WinPopup	2017-02-18 12:36:29.013936835 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Resource script for sending WinPopups using smbclient
 # derived from Alan Robertson's MailTo script
diff -ur resource-agents-3.9.7-1/heartbeat/Xen resource-agents-3.9.7/heartbeat/Xen
--- resource-agents-3.9.7-1/heartbeat/Xen	2017-02-18 12:22:23.015008442 +0300
+++ resource-agents-3.9.7/heartbeat/Xen	2017-02-18 12:36:28.991716454 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 # Support:      linux-ha@lists.linux-ha.org
diff -ur resource-agents-3.9.7-1/heartbeat/Xinetd resource-agents-3.9.7/heartbeat/Xinetd
--- resource-agents-3.9.7-1/heartbeat/Xinetd	2017-02-18 12:22:23.015133286 +0300
+++ resource-agents-3.9.7/heartbeat/Xinetd	2017-02-18 12:36:29.099681530 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 # Startup/shutdown script for services managed by xinetd.
 #
diff -ur resource-agents-3.9.7-1/heartbeat/zabbixserver resource-agents-3.9.7/heartbeat/zabbixserver
--- resource-agents-3.9.7-1/heartbeat/zabbixserver	2017-02-18 12:22:23.025949053 +0300
+++ resource-agents-3.9.7/heartbeat/zabbixserver	2017-02-18 12:36:28.981607957 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #
 #   zabbixserver OCF RA for zabbix_server daemon
diff -ur resource-agents-3.9.7-1/tools/ocf-tester.in resource-agents-3.9.7/tools/ocf-tester.in
--- resource-agents-3.9.7-1/tools/ocf-tester.in	2017-02-18 12:22:23.046593388 +0300
+++ resource-agents-3.9.7/tools/ocf-tester.in	2017-02-18 12:36:29.176393550 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #	$Id: ocf-tester,v 1.2 2006/08/14 09:38:20 andrew Exp $
 #
@@ -216,7 +216,7 @@
 test_permissions() {
     action=meta-data
     debug ${1:-"Testing permissions with uid nobody"}
-    su nobody -s /bin/sh $agent $action > /dev/null
+    su nobody -s /usr/bin/bash $agent $action > /dev/null
 }
 
 test_metadata() {
